integration/container_stats_test.go

```diff
@@ -296,6 +296,29 @@ func TestContainerListStatsWithIdSandboxIdFilter(t *testing.T) {
 			testStats(t, s, config)
 		}
 	}
+
+	t.Logf("Fetch container stats for sandbox truncID and container truncID filter ")
+	for id, config := range containerConfigMap {
+		require.NoError(t, Eventually(func() (bool, error) {
+			stats, err = runtimeService.ListContainerStats(
+				&runtime.ContainerStatsFilter{Id: id[:3], PodSandboxId: sb[:3]})
+			if err != nil {
+				return false, err
+			}
+			if len(stats) != 1 {
+				return false, fmt.Errorf("Unexpected stats length")
```

Comment:
s/Unexpected/unexpected.

And also the one above.

Reply:
done